docs/Reference/Permissioning-Types.md

```diff
@@ -0,0 +1,71 @@
+---
+description: Permissioning types reference
+---
+
+# Permissioning types
+
+This reference describes account access and status types for
+[permissioning](../Concepts/Permissioning/PermissionsOverview.md).
+
+## Account access types
+
+The following table indicates the numeric value for each account access type.
+
+|   AccessType   | Value |
```

Comment:
Could probably add a 3rd column to this table which includes it's privileges from the bullet points below.